libraries/ms-drivers/src/max6643_fan_controller.c

```diff
@@ -0,0 +1,35 @@
+#include <stddef.h>
+#include <string.h>
+
+#include "gpio.h"
+#include "gpio_it.h"
+#include "interrupt.h"
+#include "max6643_fan_controller.h"
```

Comment:
Nitpick: the `#include "max6643_fan_controller.h"` should be at the top of the file separated from the other includes by a blank line.